src/common/meta/src/ddl/drop_database/cursor.rs

```diff
@@ -248,38 +229,6 @@ mod tests {
         assert_eq!(executor.target, DropTableTarget::Physical);
     }
 
-    #[tokio::test]
-    async fn test_next_with_logical_tables() {
```

Comment:
Let's cover the case with logical tables. you can update the test instead of removing it.